lib/Model/Importer/Binary/ELFImporter.cpp

```diff
@@ -461,6 +462,9 @@ void ELFImporter<T, HasAddend>::findMissingTypes(object::ELFFile<T> &TheELF,
       revng_log(ELFImporterLog, " Importing Model for: " << DependencyLibrary);
       auto BinaryOrErr = llvm::object::createBinary(DependencyLibrary);
       if (auto Error = BinaryOrErr.takeError()) {
+        // TODO: wouldn't user want to know we couldn't find one of
+        //       the dependencies?
+        //       Sounds like we want a better way to report this.
```

Comment:
I'd say this is a candidate for diagnostics that can get to the user.

As of now, we don't have it. I suggest to introduce a new convention for logs that should be turned into diagnostic:

```
// TODO: turn into diagnostic
```